server/conf/routes

```diff
@@ -5,7 +5,11 @@
 # Home page
 GET     /                           controllers.Application.index()
 
+# Icon
+GET  /favicon.ico   controllers.Assets.at(path="/public/images", file="favicon.png")
```

Comment:
We don't need. However, remove this produces disappointing error message on the server side though it does not influence the flow.